### ISSUE TITLE
Locks disappearing & giving errors.

### ISSUE THREAD

**Issue by Raphy123** [megoesrawr/Modern-LWC#95]
**LWC Version (/lwc admin version):** 2.0.7
**Server version (/version):** 1.12.2

**Short five word summary of issue:** 

unlock automaticaly after the update; problem on some items

**Detailed description of problem:**

Some of the locks have disappeared. A big problem for our server. Since your last update some locks even ask to contact an admin.
":PLAYER_INTERACT_ when we clicked on some items locked

no error in console

**Comment by Raphy123** [comment]
A screenshot if u want to see the error msg when for example we want to turn itemframe https://zupimages.net/up/18/19/zkls.png

**Comment by Raphy123** [comment]
Error message : https://pastebin.com/Kxgu5PHS

**Comment by saturnux** [comment]
try to unlock, and relock after. if it's a bloc, a player can do that.
if it's an armorstand, only an admin of lwc can delete by : lwc admin remove <id> 
the <id> can be found by /cinfo
only remove one by one it long, but that work.

**Comment by Raphy123** [comment]
yes but that's not a solution when you have a big server

**Comment by saturnux** [comment]
i agree, but i don't have any other
.

**Comment by Chevels** [comment]
Exactly the same problem here. I tried every available version of 2.0 to 2.0.7 and that's the drama, my existing protections no longer work. They are all corrupt, or they cause errors in the console.

My server is 7 years old and I use LWC since the beginning; my LWC database has 59441 protections and weighs 56MB (sqlite). It would be dramatic for our server to have to lose all our protections!

**Will these conversion issues be resolved in a future update?**

I had to put LWC back in version 1.9.4 and put back a backup of my database...
